sql/auto_uninstall.sql

```diff
@@ -1,23 +1,38 @@
-SET FOREIGN_KEY_CHECKS=0;
-
-DELETE FROM civicrm_setting WHERE `name` LIKE 'membershipextras_paymentplan_%';
-
+-- +--------------------------------------------------------------------+
+-- | CiviCRM version 4.7                                                |
+-- +--------------------------------------------------------------------+
+-- | Copyright CiviCRM LLC (c) 2004-2018                                |
+-- +--------------------------------------------------------------------+
+-- | This file is a part of CiviCRM.                                    |
+-- |                                                                    |
+-- | CiviCRM is free software; you can copy, modify, and distribute it  |
+-- | under the terms of the GNU Affero General Public License           |
+-- | Version 3, 19 November 2007 and the CiviCRM Licensing Exception.   |
+-- |                                                                    |
+-- | CiviCRM is distributed in the hope that it will be useful, but     |
+-- | WITHOUT ANY WARRANTY; without even the implied warranty of         |
+-- | MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.               |
+-- | See the GNU Affero General Public License for more details.        |
+-- |                                                                    |
+-- | You should have received a copy of the GNU Affero General Public   |
+-- | License and the CiviCRM Licensing Exception along                  |
+-- | with this program; if not, contact CiviCRM LLC                     |
+-- | at info[AT]civicrm[DOT]org. If you have questions about the        |
+-- | GNU Affero General Public License or the licensing of CiviCRM,     |
+-- | see the CiviCRM license FAQ at http://civicrm.org/licensing        |
+-- +--------------------------------------------------------------------+
+--
+-- Generated from drop.tpl
+-- DO NOT EDIT.  Generated by CRM_Core_CodeGen
+--
 -- /*******************************************************
--- * Delete External IDs value tables
+-- *
+-- * Clean up the exisiting tables
+-- *
 -- *******************************************************/
-DROP TABLE IF EXISTS `civicrm_value_contribution_ext_id`;
```

Comment:
@16kilobyte should these lines to removed?

Reply:
They definitely should not be removed!